src/containers/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useReducer } from 'react'
+import axios from 'axios'
+import Title from '../components/Title'
+import Foot from '../components/foot'
+import Content from '../components/content'
+import { contentReducer, defaultContentState } from '../store/contentReducer'
+import '../assets/sass/App.scss'
+
+function App () {
+  const [contentState, contentDispatch] = useReducer(contentReducer, defaultContentState)
+  useEffect(() => {
+    axios.get('./content.json')
+      .then(res => {
+        contentDispatch({ type: 'GET_IPAD_INFOS_SUCCESS', payload: res.data })
+      })
+      .catch(err => {
+        contentDispatch({ type: 'GET_IPAD_INFOS_FAILED' })
+      })
+  }, [])
+
+  return (
+    <div className='app'>
+      <ContentContext.Provider value = {{ contentState, contentDispatch }}>
```

Comment:
when you dispatch (GET_IPAD_INFOS_FAILED), you will get some error.
Please display the error here.